packages/hash/frontend/src/pages/[account-slug]/types/entity-type/use-entities-table.tsx

```diff
@@ -73,7 +73,7 @@ export const useEntitiesTable = (
 
     const rows: TypeEntitiesRow[] =
       entities?.map((entity) => {
-        const entityLabel = generateEntityLabel(subgraph);
+        const entityLabel = generateEntityLabel(subgraph, entity);
```

Comment:
I believe this was appearing as a block because the `Subgraph` comes from [this call](https://github.com/hashintel/hash/blob/e2b8fff388244a1d106e9825e373a66bf21f5ac7/packages/hash/frontend/src/pages/%5Baccount-slug%5D/types/entity-type/use-entity-type-entities.tsx#L29-L45)

That call doesn't constrain based on the entity-type (maybe we should do that?), so I assume the first root was a Block by coincidence of the query

Reply:
Ah I remember why we can't do that presently. I [raised it in Slack](https://hashintel.slack.com/archives/C022217GAHF/p1669723758095149), although so far no responses.